h3/src/webtransport/session_id.rs

```diff
@@ -0,0 +1,50 @@
+use std::convert::TryFrom;
+
+use crate::proto::{
+    coding::{Decode, Encode},
+    stream::{InvalidStreamId, StreamId},
+    varint::VarInt,
+};
+
+/// Identifies a WebTransport session
+///
+/// The session id is the same as the stream id of the CONNECT request.
+#[derive(Debug, Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash)]
+pub struct SessionId(pub u64);
```

Comment:
I recommend that the field should be private.

Reply:
Done